modules/http-api/src/main/java/org/mule/runtime/http/api/sse/SseEndpointManager.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 Salesforce, Inc. All rights reserved.
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.runtime.http.api.sse;
+
+import org.mule.api.annotation.Experimental;
+import org.mule.api.annotation.NoImplement;
+
+/**
+ * Object to manage an SSE Endpoint. It allows temporarily disabling it or removing it from the server.
+ * <p>
+ * This API is EXPERIMENTAL. Do not use it until it is stable.
+ */
+@Experimental
+@NoImplement
+public interface SseEndpointManager {
```

Comment:
looking at where this is returned, should this better be called something like `SseServer`?

Reply:
This is similar to the RequestHandlerManager. It is an object that can be used to enable/disable/remove the endpoint from the server.
(A "disabled" handler returns 503-Unavailable)